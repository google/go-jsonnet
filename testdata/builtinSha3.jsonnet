std.sha3("foo")