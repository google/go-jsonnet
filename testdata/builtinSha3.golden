"4bca2b137edc580fe50a88983ef860ebaca36c857b1f492839d6d7392452a63c82cbebc68e3b70a2a1480b4bb5d437a7cba6ecf9d89f9ff3ccd14cd6146ea7e7"
